src/main/webapp/js/community-page-loader.js

```diff
@@ -39,16 +44,46 @@
 		
         if(aboutMe)
         {
-          const aboutMeText = document.createElement('textarea');
-          aboutMeText.setAttribute('readonly', 'readonly');
-          aboutMeText.innerHTML = aboutMe;
-          userListItem.appendChild(aboutMeText);
+          const aboutMeDiv = document.createElement('Div');
+          aboutMeDiv.innerHTML = aboutMe;
+          userListItem.appendChild(aboutMeDiv);
         }
         return userListItem;
       }
+	  
+	  /*filers users by key word in their name or aboutMe description*/
+	  function filterUsers() {
+		const keyword = document.getElementById('search-text').value;
+		const listul = document.getElementById('list');
+        listul.innerHTML = '';
+		var filteredList = [];
+		
+		if(userNameAndAboutList.length!=0)
+		{
+           if(keyword=="" || keyword===undefined)
+		   {
+             fetchUserList();
```

Comment:
does this return the entire list when the keyword is empty? (im sorry just making sure i understand it)

Reply:
Yes it does, so if the search textarea is empty, it will return the name of all users.

Reply:
because when search area is empty that means user wants to see everything, and they don't want to filter the list